testsuite/MDAnalysisTests/lib/test_distances.py

```diff
@@ -321,11 +367,24 @@ class FakeArray(np.ndarray):
 def DCD_Universe():
     universe = MDAnalysis.Universe(PSF, DCD)
     trajectory = universe.trajectory
+    return universe, trajectory
+
+# second independent universe required for 
+# TestDistanceArrayDCD_TRIC.test_atomgroup_simple
+@pytest.fixture()
+def DCD_Universe2():
+    universe = MDAnalysis.Universe(PSF, DCD)
+    trajectory = universe.trajectory
```

Comment:
Just a comment: For all of these fixtures, we could have just written
```python
    return universe, universe.trajectory
```
although it's a bit superfluous to return `trajectory` when calling code can just get it as easily (`U.trajectory` is not much more work than `trajectory` and it's also clearer); furthermore, the fixture name is also confusing because it is not just the universe but also the trajectory.

None of this is your fault, of course, I'm just commenting while looking at the code.

Reply:
Would you like me to fix it here, raise an issue or leave it for now?

Reply:
Up to you — it would make for an outreachy-type issue.